DOCSP-46812 - Distribute Type Hints (#167)
mongodb/docs-pymongo: source/includes/type-hints/troubleshooting-incompatible-type.rst

Incompatible Type
~~~~~~~~~~~~~~~~~

> If you use the generic form of the ``MongoClient`` class, your type checker might
> show an error similar to the following:

Reviewer: I'm a little weak on Python, but I think this is more specific:
```suggestion
If you do not specify a type when you instantiate the ``MongoClient`` class, your type checker might
show an error similar to the following:
```

Author: I agree about keeping 'generic' as the term for 'type that accepts another type as an argument'
